cli/crates/backend/assets/default.env

```diff
@@ -0,0 +1 @@
+# KEY=VALUE
```

Comment:
?

Reply:
That's the default env file we'll output, it's a comment